distro.py

```diff
@@ -738,8 +747,9 @@ def __init__(
                 self.os_release_file = usr_lib_os_release_file
 
         self.distro_release_file = distro_release_file or ""  # updated later
-        self.include_lsb = include_lsb
-        self.include_uname = include_uname
+        self.include_lsb = include_lsb if not root_dir else False
+        self.include_uname = include_uname if not root_dir else False
+        self.include_third_bins = include_third_bins if not root_dir else False
```

Comment:
I see some conflict here. What if we get passed say `include_uname=True, include_third_bins=False`? Maybe raise `ValueError` with helpful text whenever `(include_lsb or include_uname) and not include_third_bins` holds true?

Reply:
Actually this proposal implied that `include_third_bins` does not "target" `lsb_release` and `uname` calls. See next comment for another possible "fix".